content/docs/use-cases/datasets-cache.md

```diff
@@ -0,0 +1,107 @@
+# Datasets Cache
+
+Datasets used in data science tend to exceed typical storage and computing
+capacities. Valuable time is wasted waiting for downloads, and the wait repeats
+across environments. Combined storage needs expand rapidly as more people and
+projects acquire the same data, creating duplication and increasing cost. Is
+there an effective way to perform this process?
+
+![](/img/storage-layers.png) _Data storage infrastructure_
+
+DVC's built-in data <abbr>caching</abbr> enables you to implement a simple and
+efficient storage layer globally — for all your team's projects. This approach
+can help in situations such as:
+
+- You need to speed up data transfers from a massive object store on the cloud,
```

Comment:
```suggestion
- You need to minimize data transfers from a massive object store on the cloud,
```
Thoughts on this? Current wording makes me think DVC will somehow increase network speed.

Reply:
> Current wording makes me think DVC will somehow increase network speed

Implementing the pattern we're suggesting will increase speed. Not of the network of course but by bringing the data closer. The suggestion works too but I think "speed" is a keyword in this doc.

Reply:
Changing to "from a massive object which is currently on the cloud" for now